reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,67 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.21.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+
+* Initial support for installing Anbox Cloud Appliance on Ubuntu 24.04.<!--AC-2229-->
+* GL Async swap support is disabled by default. See [GL Async swap support](https://discourse.ubuntu.com/t/ams-configuration/20872#gl-async-swap-support-11) for more information.<!--AC-2228-->
+* Production support for Vulkan on NVIDIA GPUs is available:
+  - The default graphics driver stack used is Vulkan, VirGL/Venus and ANGLE. Latency and flickering is improved for VirGL by introducing a dedicated component, frame renderer. <!--AC-2152-->
+  - System tracing is enabled for Android 12 and 13. This helps in improving the performance of your application. See [System tracing overview](https://developer.android.com/topic/performance/tracing) for more information. <!--AC-2161-->
```

Comment:
I feel like we need a dedicated page to introduce this new feature like what we did in our internal doc
https://github.com/canonical/anbox-internal-docs/blob/main/howto/anbox/android-tracing.md
Because currently, Android system tracing support is gated by the `android.enable_tracing` feature flag. Route the end user to  [System tracing overview](https://developer.android.com/topic/performance/tracing) without enabling this feature may not help

Reply:
I think we can but it might require some wording adjustments.
@morphis What do you think?

Reply:
Wording adjustments are fine, I was asking about technical accuracy. I will look to use the internal docs as a base and create a separate PR for that.

Reply:
I created a PR for this: https://github.com/canonical/anbox-cloud-docs/pull/34

Reply:
@keirthana, @morphis  and I had a quick discussion about it. Let's include that in the release announcement but hold off on creating a dedicated page to introduce this feature, as it may require some polishing to improve the overall user experience for Android system tracing

Reply:
okay, I will then leave the PR on that open for now and will just file it for later once I am back.

Reply:
Confirmed not to add this for now based on Simon's comments on https://github.com/canonical/anbox-cloud-docs/pull/34